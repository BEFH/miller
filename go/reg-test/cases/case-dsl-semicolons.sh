run_mlr --from $indir/s.dkvp --idkvp --opprint put '$x=1; $y=2'
run_mlr --from $indir/s.dkvp --idkvp --opprint put '$x=1; $y=2;'
run_mlr --from $indir/s.dkvp --idkvp --opprint put '; $x=1'
run_mlr --from $indir/s.dkvp --idkvp --opprint put ';;;;;'
run_mlr --from $indir/s.dkvp --idkvp --opprint put 'begin{} $x=1; end{}'
run_mlr --from $indir/s.dkvp --idkvp --opprint put 'begin{}; $x=1; end{}'
run_mlr --from $indir/s.dkvp --idkvp --opprint put 'begin{} $x=1;;; end{}'
run_mlr --from $indir/s.dkvp --idkvp --opprint put ';;;begin{} ;;; end{};;'
