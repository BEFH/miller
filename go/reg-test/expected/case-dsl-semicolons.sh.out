
mlr --from ./reg-test/input/s.dkvp --idkvp --opprint put $x=1; $y=2
a   b   i x y
pan pan 1 1 2
eks pan 2 1 2
wye wye 3 1 2
eks wye 4 1 2

mlr --from ./reg-test/input/s.dkvp --idkvp --opprint put $x=1; $y=2;
a   b   i x y
pan pan 1 1 2
eks pan 2 1 2
wye wye 3 1 2
eks wye 4 1 2

mlr --from ./reg-test/input/s.dkvp --idkvp --opprint put ; $x=1
a   b   i x y
pan pan 1 1 0.7268028627434533
eks pan 2 1 0.5221511083334797
wye wye 3 1 0.33831852551664776
eks wye 4 1 0.13418874328430463

mlr --from ./reg-test/input/s.dkvp --idkvp --opprint put ;;;;;
a   b   i x                   y
pan pan 1 0.3467901443380824  0.7268028627434533
eks pan 2 0.7586799647899636  0.5221511083334797
wye wye 3 0.20460330576630303 0.33831852551664776
eks wye 4 0.38139939387114097 0.13418874328430463

mlr --from ./reg-test/input/s.dkvp --idkvp --opprint put begin{} $x=1; end{}
a   b   i x y
pan pan 1 1 0.7268028627434533
eks pan 2 1 0.5221511083334797
wye wye 3 1 0.33831852551664776
eks wye 4 1 0.13418874328430463

mlr --from ./reg-test/input/s.dkvp --idkvp --opprint put begin{}; $x=1; end{}
a   b   i x y
pan pan 1 1 0.7268028627434533
eks pan 2 1 0.5221511083334797
wye wye 3 1 0.33831852551664776
eks wye 4 1 0.13418874328430463

mlr --from ./reg-test/input/s.dkvp --idkvp --opprint put begin{} $x=1;;; end{}
a   b   i x y
pan pan 1 1 0.7268028627434533
eks pan 2 1 0.5221511083334797
wye wye 3 1 0.33831852551664776
eks wye 4 1 0.13418874328430463

mlr --from ./reg-test/input/s.dkvp --idkvp --opprint put ;;;begin{} ;;; end{};;
a   b   i x                   y
pan pan 1 0.3467901443380824  0.7268028627434533
eks pan 2 0.7586799647899636  0.5221511083334797
wye wye 3 0.20460330576630303 0.33831852551664776
eks wye 4 0.38139939387114097 0.13418874328430463

